bugbug/model.py

```diff
@@ -581,8 +596,13 @@ def train(self, importance_cutoff=0.15, limit=None):
 
     @staticmethod
     def load(model_file_name: str) -> "Model":
-        with open(model_file_name, "rb") as f:
-            return pickle.load(f)
+        with open(os.path.join(model_file_name, "pickled.pkl"), "rb") as f:
```

Comment:
It will be cleaner to store the file path in a variable. Less nesting:
```suggestion
        model_path = os.path.join(model_file_name, "model.pkl")
        with open(model_path, "rb") as f:
```